draft-ietf-rats-corim.md

```diff
@@ -387,8 +387,18 @@ The following describes each child element of this type.
 
 ### Profile Types {#sec-corim-profile-types}
 
-A profile specifies which of the optional parts of a CoRIM are required, which
-are prohibited and which extension points are exercised and how.
+Profiling is the mechanism that allows the base CoRIM schema to be customised to fit a specific Attester or Attesters' family.
```

Comment:
```suggestion
Profiling is the mechanism that allows the base CoRIM schema to be customised to fit a specific Attester or Attesters' class.
```

Reply:
I'm OK removing "Attesters' family" altogether.
In alternative, use Ned's suggestion in https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/76#discussion_r1154636121

Reply:
I'm going with option 1